src/export.rs

```diff
@@ -0,0 +1,122 @@
+use anyhow::{Result, Context, anyhow};
+use clap::Args;
+use std::path::Path;
+use crate::{
+    account::derive_and_cache_addresses,
+    utils::{display_string_discreetly, load_wallet},
+    DEFAULT_CACHE_ACCOUNTS,
+};
+
+#[derive(Debug, Args)]
+pub struct Export {
+    /// Forces export even if it might be unsafe
+    #[clap(short, long)]
+    pub force: bool,
+    /// How many accounts to cache by default (Default 10)
+    #[clap(short, long)]
+    pub cache_accounts: Option<usize>,
```

Comment:
Let's use clap to set the default
```suggestion
    /// The number of accounts to cache
    #[clap(long, default_value = 10)]
    pub cache_accounts: usize,
```